normandy/base/api/authentication.py

```diff
@@ -81,11 +82,19 @@ def authenticate_header(self, request):
 
     @backoff.on_exception(backoff.constant, requests.exceptions.RequestException, max_tries=5)
     def fetch_oidc_user_profile(self, access_token):
+        cache_key = f"oidc-profile-{access_token}"
```

Comment:
Is it safe to use the access token as the cache key? I'm not sure what else we would use though. Maybe only a portion of it, like the first half of the token (16 chars)?

Reply:
or i can just hash it.